From d6d0b534a12bd3e34e5070b2a40c21e67fc2bcdc Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 29 Mar 2023 17:36:11 +0200
Subject: [PATCH 590/954] feat(docs): add STM32MP2 docs links

Add links to official STMicroelectronics documentation (STM32MP2
series presentation and wiki).

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 3887400c93d77913c9ba0cae0736a01a72252544 feat(docs): add STM32MP2 docs links]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I2fca0da56bc6064c222df34493921dff3e119a22
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/373835
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 docs/plat/st/stm32mp2.rst | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/docs/plat/st/stm32mp2.rst b/docs/plat/st/stm32mp2.rst
index ef0f03eb72..92cb8b6b4f 100644
--- a/docs/plat/st/stm32mp2.rst
+++ b/docs/plat/st/stm32mp2.rst
@@ -4,6 +4,8 @@ STM32MP2
 STM32MP2 is a microprocessor designed by STMicroelectronics
 based on Arm Cortex-A35.
 
+More information can be found on `STM32MP2 Series`_ page.
+
 For TF-A common configuration of STM32 MPUs, please check
 :ref:`STM32 MPUs` page or `Github STM32 MPUs`_.
 
@@ -61,6 +63,8 @@ Each line comes with a security option (cryptography & secure boot) and a Cortex
 - D      Basic + Cortex-A35 @ 1.5GHz
 - F      Secure Boot + HW Crypto + Cortex-A35 @ 1.5GHz
 
+The `STM32MP2 part number codification`_ page gives more information about part numbers.
+
 Memory mapping
 --------------
 
@@ -234,6 +238,8 @@ __________________
         --soc-fw build/stm32mp2/release/bl31.bin \
         build/stm32mp2/release/fip.bin
 
+.. _STM32MP2 Series: https://www.st.com/en/microcontrollers-microprocessors/stm32mp2-series.html
+.. _STM32MP2 part number codification: https://wiki.st.com/stm32mpu/wiki/STM32MP25_microprocessor#Part_number_codification
 .. _Github STM32 MPUs: https://github.com/STMicroelectronics/arm-trusted-firmware/tree/HEAD/docs/plat/st/stm32mpus.rst
 .. _STMicroelectronics DDR PHY github: https://github.com/STMicroelectronics/stm32-ddr-phy-binary
 
-- 
2.53.0

